sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/MultiValueStringOperatorConversions.java

```diff
@@ -169,9 +168,9 @@ public SqlOperator calciteOperator()
     }
 
     @Override
-    protected String getFilterExpression(List<DruidExpression> druidExpressions)
+    public String getDruidFunctionName()
```

Comment:
should this still extend ArrayContainsOperatorConversion? I guess i see the benefit of overlapping filter conversion, but given that it uses some other native filters, such as `ArrayContainsElementFilter`, is that ok?